src/Rating.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace CanadaPost;
+
+use LSS\Array2XML;
```

Comment:
Either use or not include it. If you want you can leave it for later, but it should be straightforward:

```
$content = [
    'customer-number' => $this->customerNumber,
    'parcel-characteristics' => [
        'weight' => $weight,
    ],
    'origin-postal-code' => $originPostalCode,
    'destination' => [
        'domestic' => [
            'postal-code' => $postalCode,
        ],
    ],
];
$xml = Array2XML::createXML('mailing-scenario', $content);
$envelope = $xml->documentElement;
$envelop->setAttribute('xmlns', 'http://www.canadapost.ca/ws/ship/rate-v3');
$payload = $xml->saveXML();
```